app/services/Config.scala

```diff
@@ -27,9 +27,9 @@ sealed trait Config {
 
   def capiUrl: String = getRequiredStringProperty("capi.url")
   def capiKey: String = getRequiredStringProperty("capi.key")
-  def capiPreviewUrl: String = getRequiredStringProperty("capi.preview.url")
-  def capiPreviewUser: String = getRequiredStringProperty("capi.preview.username")
-  def capiPreviewPassword: String = getRequiredStringProperty("capi.preview.password")
+  def capiPreviewIAMUrl: String = getRequiredStringProperty("capi.preview.iamUrl")
+  def capiPreviewRole: String = getRequiredStringProperty("capi.preview.role")
```

Comment:
Do both these new variables have to be in the "hidden" S3 config or can they be in the regular in-code config?

Particularly the role name

Reply:
the role arn is effectively replacing the username/password - I've been handling it as 'secret' config so far